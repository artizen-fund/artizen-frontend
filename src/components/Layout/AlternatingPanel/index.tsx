import React from 'react'
import styled from 'styled-components'
import { assetPath } from '@lib'
import { breakpoint, typography } from '@theme'

export interface IAlternatingPanel {
  image: string
  imageDark?: string
  title: string
  copy: string
  imageOnRight?: boolean
  destination?: string
  buttonLabel?: string
  children?: React.ReactElement
}

const AlternatingPanel = ({ image, imageDark, title, copy, imageOnRight, children }: IAlternatingPanel) => {
  return (
    <Wrapper {...{ imageOnRight }}>
      <Image {...{ image, imageDark }} />
      <Content>
        <Title>{title}</Title>
        <Copy>{copy}</Copy>
        {children}
      </Content>
    </Wrapper>
  )
}

const Wrapper = styled.div<Pick<IAlternatingPanel, 'imageOnRight'>>`
  display: flex;
  flex-direction: column;
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    flex-direction: ${props => (props.imageOnRight ? 'row' : 'row-reverse')};
    align-items: center;
  }
`

const Image = styled.div<Pick<IAlternatingPanel, 'image' | 'imageDark'>>`
  width: 100%;
  height: 54.56vw;
  background-repeat: no-repeat;
  background-size: contain;
  background-position: center center;
  background-image: url(${props => assetPath(props.image)});
  ${props =>
    props.imageDark &&
    `
    @media (prefers-color-scheme: dark) {
      background-image: url(${assetPath(props.imageDark)});
    }
  `}
  @media only screen and (min-width: ${breakpoint.tablet}px) {
    width: 100vw;
    height: 27.28vw;
  }
`

const Content = styled.div`
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  gap: 15px;
  padding: 40px 0;
  @media only screen and (min-width: ${breakpoint.tablet}px) {
    min-width: 50%;
    padding: 0 45px;
  }
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    padding: 0 65px;
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    padding: 0 80px;
  }
`

const Title = styled.h2`
  ${typography.title.l2}
`

const Copy = styled.p`
  ${typography.body.l2}
`

export default AlternatingPanel
