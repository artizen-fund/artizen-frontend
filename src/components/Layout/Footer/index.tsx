import { useState } from 'react'
import styled from 'styled-components'
import { breakpoint, palette, typography } from '@theme'
import { rgba } from '@lib'
import { Button, PagePadding } from '@components'
import SocialLinks from './SocialLinks'

const Footer = () => {
  return (
    <>
      <PagePadding black>
        <Wrapper>
          <Rule gridSpace="rule1" />
          <BigLinks>
            <li>Mission</li>
            <li>Leaderboard</li>
            <li>How it Works</li>
            <li>Community</li>
          </BigLinks>
          <SmallLinks>
            <li>Contact</li>
            <li>Careers</li>
            <li>Press</li>
          </SmallLinks>
          <Rule gridSpace="rule2" />
          <CTA>
            <Copy>Join the mission to fund public goods.</Copy>
            <Buttons>
              <Button onClick={() => alert('coming soon')} outline size="l1">
                Donate
              </Button>
              <Button onClick={() => alert('coming soon')} outline size="l1">
                Sign In
              </Button>
            </Buttons>
          </CTA>
          <Rule gridSpace="rule3" />
          <SocialLinks />
        </Wrapper>
      </PagePadding>
      <StyledPagePadding>
        <Credits>
          <div>Open source platform made with ❤️ by a globally distributed team</div>
          <ul>
            <li>Privacy Policy</li>
            <li>Terms &amp; Conditions</li>
          </ul>
        </Credits>
      </StyledPagePadding>
    </>
  )
}

const Wrapper = styled.footer`
  display: grid;
  grid-gap: 18px;
  grid-template-areas:
    'rule1 rule1'
    'bigLinks smallLinks'
    'rule2 rule2'
    'cta cta'
    'rule3 rule3'
    'social social';
  grid-template-columns: repeat(2, 1fr);
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    grid-gap: 20px 40px;
    grid-template-areas:
      'rule1 rule1 rule2'
      'bigLinks smallLinks cta'
      'rule3 rule3 rule3'
      'social social social';
    grid-template-columns: 2fr 1fr 3fr;
    /* todo: this grid template is not getting us equal columns */
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    grid-gap: 30px 65px;
  }
  color: ${rgba(palette.white)};
`

const Rule = styled.hr<{ gridSpace: string }>`
  grid-area: ${props => props.gridSpace};
  display: block;
  appearance: none;
  width: 100%;
  height: 1px;
  background-color: ${rgba(palette.barracuda)};
  border: 0px;
  outline: 0px;
`

const BigLinks = styled.ul`
  grid-area: bigLinks;
  ${typography.title.l2}
`

const SmallLinks = styled.ul`
  grid-area: smallLinks;
  ${typography.label.l0}
`

const CTA = styled.div`
  grid-area: cta;
`

const Copy = styled.div`
  ${typography.title.l2}
`

const Buttons = styled.div`
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    display: flex;
    flex-direction: row;
    gap: 10px;
  }
`

const Credits = styled.footer`
  text-align: center;
  ${typography.label.l2}
  color: ${rgba(palette.barracuda)};
  ul {
    display: flex;
    justify-content: space-around;
    li {
      display: block;
    }
  }
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
  }
`

const StyledPagePadding = styled(props => <PagePadding {...props} />)`
  padding: 15px 0;
  background-color: ${rgba(palette.white)};
  @media (prefers-color-scheme: dark) {
    background-color: ${rgba(palette.black)};
    color: ${rgba(palette.white)};
  }
`

export default Footer
