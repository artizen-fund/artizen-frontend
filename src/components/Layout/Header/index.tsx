import { useState } from 'react'
import Link from 'next/link'
import styled from 'styled-components'
import { useScrollPosition } from '@n8tb1t/use-scroll-position'
import { Button } from '@components'
import AccountButton from './AccountButton'
import SessionShelf from './SessionShelf'
import HowItWorks from './HowItWorks'
import Shelf from './Shelf'
import { breakpoint, palette, glyphKey } from '@theme'
import { rgba } from '@lib'

type ShelfType = 'session' | 'howItWorks'

const Header = () => {
  const [shadowVisible, setShadowVisible] = useState(false)
  useScrollPosition(({ currPos }) => setShadowVisible(currPos.y > 0), [], undefined, true, 50)

  const [navVisible, setNavVisible] = useState(false)

  const [visibleShelf, setVisibleShelf] = useState<ShelfType>()
  const toggleShelf = (shelf?: ShelfType) => setVisibleShelf(shelf === visibleShelf ? undefined : shelf)

  return (
    <>
      <Wrapper {...{ shadowVisible }} className={visibleShelf ? 'visibleShelf' : ''}>
        <Items>
          <Link href="/">
            <a>
              <Logo>Artizen</Logo>
            </a>
          </Link>
          <MobileNavButton
            onClick={() => setNavVisible(!navVisible)}
            icon={glyphKey.arrow}
            iconOnRight
            outline
            size="l1"
          >
            Menu
          </MobileNavButton>
        </Items>
        <Items>
          <Nav>
            <ul>
              <li>
                <Link href="/">Leaderboard</Link>
              </li>
              <li onClick={() => toggleShelf('howItWorks')}>How it Works</li>
            </ul>
          </Nav>
          <Button href="/" glyph={glyphKey.donate} level={1}>
            Donate
          </Button>
          <AccountButton loggedOutAction={() => toggleShelf('session')} />
        </Items>
      </Wrapper>
      <Shelf visible={visibleShelf === 'session'} hideShelf={() => toggleShelf()} {...{ shadowVisible }}>
        <SessionShelf />
      </Shelf>
      <Shelf visible={visibleShelf === 'howItWorks'} hideShelf={() => toggleShelf()} {...{ shadowVisible }}>
        <HowItWorks />
      </Shelf>
    </>
  )
}

const Wrapper = styled.header<{ shadowVisible: boolean }>`
  position: fixed;
  z-index: 102;
  top: 0;
  left: 0;

  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;

  padding: 0 40px;
  width: 100%;
  height: 64px;
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    height: 72px;
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    height: 88px;
  }

  background: ${props => rgba(palette.white, props.shadowVisible ? 0.98 : 1)};
  @media (prefers-color-scheme: dark) {
    background: ${props => rgba(palette.slate, props.shadowVisible ? 0.98 : 1)};
  }
  filter: drop-shadow(
    ${props => (props.shadowVisible ? '0px 4px 16px rgba(0, 0, 0, 0.48)' : '0px 0.5px 0px rgba(217, 219, 224, 1)')}
  );
  backdrop-filter: blur(${props => (props.shadowVisible ? 16 : 0)}px);

  border-bottom: 0.5px solid transparent;
  transition: border-color 0.3s 0.15s ease-in-out, background-color 0.3s ease-in-out, filter 0.3s ease-in-out,
    backdrop-filter 0.3s ease-in-out;
  &.visibleShelf {
    filter: drop-shadow(0px 0.5px 0px rgba(217, 219, 224, 1));
    border-color: ${rgba(palette.stone)};
    @media (prefers-color-scheme: dark) {
      border-color: ${rgba(palette.barracuda, 0.64)};
    }
    transition: border-color 0.3s ease-in-out, background-color 0.3s ease-in-out, filter 0.3s ease-in-out,
      backdrop-filter 0.3s ease-in-out;
  }
`

const Logo = styled.div`
  display: flex;
  flex-direction: row;
  align-items: center;

  font-size: 13px;
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    font-size: 15px;
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    font-size: 16px;
  }
  text-transform: uppercase;

  &:before {
    content: ' ';
    display: block;
    width: 40px;
    height: 40px;
    @media only screen and (min-width: ${breakpoint.laptop}px) {
      height: 44px;
      width: 44px;
    }
    @media only screen and (min-width: ${breakpoint.desktop}px) {
      height: 48px;
      width: 48px;
    }
    background-color: ${rgba(palette.algae)};
    border-radius: 9999px;
    margin-right: 8px;
  }
`

const Items = styled.div`
  display: flex;
  flex-direction: row;
  align-items: center;
  gap: 20px;
`

const Nav = styled.div`
  display: none;
  @media only screen and (min-width: ${breakpoint.tablet}px) {
    display: contents;
  }
  ul {
    display: flex;
    flex-direction: row;
    align-items: center;
    gap: 20px;
    white-space: nowrap;
    li {
      cursor: pointer;
    }
  }
`

const MobileNavButton = styled(props => <Button {...props} />)`
  @media only screen and (min-width: ${breakpoint.tablet}px) {
    display: none !important;
  }
`

export default Header
