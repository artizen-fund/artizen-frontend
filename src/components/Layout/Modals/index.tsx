import { useContext, useEffect, useState } from 'react'
import styled from 'styled-components'
import { CloseButton, PostDonationData, Share, AlertModal, MediaModal, LoginModal } from '@components'
import { LayoutContext } from '@lib'
import { breakpoint } from '@theme'

const ANIMATION_TIMING = 0.6 // seconds

const Modals = () => {
  const { visibleModal, toggleModal } = useContext(LayoutContext)
  /* In order for the modal to have any kind of fade-out effect, it needs
   *   to persist even when visibleModal is changed in the LayoutContext.
   *
   * So rather than just test and switch visibleModal, we're going to use
   *   that to set a local modal state, which gets unset after transitions
   *   are concluded.
   */
  const [displayedVisibleModal, setDisplayedVisibleModal] = useState<ModalType>()
  const [visible, setVisible] = useState(false)
  useEffect(() => {
    if (!visibleModal) {
      setVisible(false)
      setTimeout(() => setDisplayedVisibleModal(undefined), ANIMATION_TIMING * 1000)
    } else {
      setVisible(true)
      setDisplayedVisibleModal(visibleModal)
    }
  }, [visibleModal])

  const renderSwitch = (visibleModal?: ModalType) => {
    switch (displayedVisibleModal) {
      case 'login':
        return <LoginModal />
      case 'postDonationData':
        return <PostDonationData />
      case 'share':
        return <Share />
      case 'alert':
        return <AlertModal headline="derp" message="herp derp derp" />
      case 'media':
        return <MediaModal />
      default:
        return <></>
    }
  }

  return (
    <Wrapper {...{ visible }}>
      <Content>
        {renderSwitch(visibleModal)}
        <CloseButton onClick={() => toggleModal?.()} {...{ visible }} />
      </Content>
    </Wrapper>
  )
}

const Content = styled.div`
  position: relative;
  pointer-events: all;
`

const Wrapper = styled.div<{ visible?: boolean }>`
  position: fixed;
  z-index: 103;
  left: 0;
  width: 100%;

  top: 64px;
  height: calc(100vh - 64px);
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    top: 72px;
    height: calc(100vh - 72px);
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    top: 88px;
    height: calc(100vh - 88px);
  }
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;

  transition: opacity ${ANIMATION_TIMING}s ease-in-out, transform ${ANIMATION_TIMING}s ease-in-out;
  opacity: ${props => (props.visible ? 1 : 0)};
  transform: translateY(${props => (props.visible ? 0 : '100px')};);
  pointer-events: none;
`

export default Modals
