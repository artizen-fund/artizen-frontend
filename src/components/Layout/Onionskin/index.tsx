import styled from 'styled-components'

const Onionskin = styled.div`
  position: fixed;
  z-index: 100;
  left: 0;
  top: 0;
  width: 100%;
  height: 100vh;

  background: rgba(0, 0, 0, 0.5);
  opacity: 0;
  transition: opacity 0.15s 0s ease-in-out;
  pointer-events: none;
  &.visible {
    opacity: 1;
    pointer-events: all;
    transition: opacity 1.5s 0.15s ease-in-out;
  }
  will-change: opacity;
`

export default Onionskin
