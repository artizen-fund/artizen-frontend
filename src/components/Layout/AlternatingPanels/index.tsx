import { cloneElement, Children } from 'react'
import styled from 'styled-components'
import { IAlternatingPanel } from '../AlternatingPanel'
import { PagePadding } from '@components'
import { breakpoint } from '@theme'

interface IAlternatingPanels {
  children: Array<React.ReactElement<IAlternatingPanel>>
}

const AlternatingPanels = ({ children }: IAlternatingPanels) => (
  <PagePadding>
    <Wrapper>{Children.map(children, (child, i) => cloneElement(child, { imageOnRight: !!(i % 2) }))}</Wrapper>
  </PagePadding>
)

const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  @media only screen and (min-width: ${breakpoint.tablet}px) {
    gap: 50px;
  }
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    gap: 65px;
  }
  @media only screen and (min-width: ${breakpoint.desktop}px) {
    gap: 85px;
  }
`

export default AlternatingPanels
