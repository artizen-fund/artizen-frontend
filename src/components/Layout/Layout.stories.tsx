import Layout from './'

export default {
  title: 'layout/Layout',
  component: Layout,
}

export const LayoutComponent = () => {
  return <Layout />
}
