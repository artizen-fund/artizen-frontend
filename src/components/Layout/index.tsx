import Head from 'next/head'
import styled from 'styled-components'
import { breakpoint } from '@theme'
import Header from './Header'
import Footer from './Footer'
import Onionskin from './Onionskin'
import { metatags as copy } from '@copy/common'
import { assetPath } from '@lib'

interface LayoutProps {
  children: React.ReactNode
}

const Layout = ({ children }: LayoutProps) => {
  return (
    <Wrapper>
      <Head>
        <title>{copy.title}</title>
        <meta charSet="UTF-8" />
        <meta name="viewport" content="width=device-width, user-scalable=no" />

        <meta property="og:type" content="website" />

        <link rel="preconnect" href="https://use.typekit.net" />
        <link rel="stylesheet" href="https://use.typekit.net/wwx1oja.css" />

        <meta property="og:title" content={copy.title} key="title" />
        <meta name="twitter:title" content={copy.title} />

        <meta property="og:url" content={copy.url} />

        <meta name="description" content={copy.description} />
        <meta name="twitter:description" content={copy.description} />
        <meta property="og:description" content={copy.description} />

        <meta name="keywords" content={copy.keywords} />

        <meta property="og:image" content={assetPath('/assets/metatags/social-share-image.jpg')} />
        <meta name="twitter:image" content={assetPath('/assets/metatags/social-share-image.jpg')} />

        <meta name="twitter:site" content={copy.twitterHandle} />
        <meta name="twitter:card" content="summary_large_image" />

        <link id="icon" rel="icon" href={assetPath('/assets/metatags/favicon-32x32.png')} />
        <link rel="mask-icon" href={assetPath('/assets/metatags/safari-pinned-tab.svg')} color="#1ACC6C" />
        <meta name="theme-color" content="#1ACC6C" />
      </Head>
      <Header />
      <Main>{children}</Main>
      <Footer />
      <Onionskin />
    </Wrapper>
  )
}

const Wrapper = styled.div``

const Main = styled.main`
  padding-top: 64px;
  min-height: 75vh;
  @media only screen and (min-width: ${breakpoint.laptop}px) {
    padding-top: 72px;
    min-height: calc(100vh - 595px);
  }
`

export default Layout
