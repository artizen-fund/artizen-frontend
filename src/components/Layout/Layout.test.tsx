import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { SessionProvider } from '@lib'
import Layout from './'

describe('Layout', () => {
  it('renders Layout unchanged', () => {
    const { container } = render(
      <SessionProvider>
        <Layout />
      </SessionProvider>,
    )
    expect(container).toMatchSnapshot()
  })
})
