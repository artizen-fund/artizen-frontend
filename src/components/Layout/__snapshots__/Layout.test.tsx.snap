// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders Layout unchanged 1`] = `
<div>
  <div
    class="sc-29d73b66-0"
  >
    <header
      class="sc-3d3423c2-0 spKTh"
    >
      <div
        class="sc-3d3423c2-2 hVsTkp"
      >
        <a
          href="/"
        >
          <div
            class="sc-3d3423c2-1 gtrDXz"
          >
            Artizen
          </div>
        </a>
        <button
          class="sc-21c50e09-2 fySYKP sc-3d3423c2-4 fIspEB "
        >
          <span>
            Menu
          </span>
          <div
            class="sc-8737affa-0 gPfcJB sc-21c50e09-0 gQPwQA"
            color="white"
          >
            arrow
          </div>
        </button>
      </div>
      <div
        class="sc-3d3423c2-2 hVsTkp"
      >
        <div
          class="sc-3d3423c2-3 jNfbAw"
        >
          <ul>
            <li>
              <a
                href="/"
              >
                Leaderboard
              </a>
            </li>
            <li>
              <a
                href="/"
              >
                How it Works
              </a>
            </li>
          </ul>
        </div>
        <a
          class="sc-21c50e09-1 irfFLJ undefined "
        >
          <div
            class="sc-8737affa-0 kwTYSl sc-21c50e09-0 gQPwQA"
            color="white"
          >
            donate
          </div>
          <span>
            Donate
          </span>
        </a>
        <div
          class="sc-47dc7821-0 bERdPH"
        >
          <div
            class="sc-47dc7821-1 gRBtCJ"
          />
          <div
            class="sc-8737affa-0 bcnHmJ sc-47dc7821-2 etaqjK"
            color="night"
          >
            hamburger
          </div>
          <div
            class="sc-47dc7821-3 ksJggS"
          >
            <div
              class="sc-47dc7821-4 kGTxhi"
            >
              RP
            </div>
          </div>
        </div>
      </div>
    </header>
    <main
      class="sc-29d73b66-1 gEpBEx"
    />
    <div
      class="sc-9199b4ac-0 gjUYWF"
    >
      <footer
        class="sc-f5137ace-0 hwwInd"
      >
        <hr
          class="sc-f5137ace-1 Gzpnn"
        />
        <ul
          class="sc-f5137ace-2 elBpRt"
        >
          <li>
            Mission
          </li>
          <li>
            Leaderboard
          </li>
          <li>
            How it Works
          </li>
          <li>
            Community
          </li>
        </ul>
        <ul
          class="sc-f5137ace-3 hQwgsl"
        >
          <li>
            Contact
          </li>
          <li>
            Careers
          </li>
          <li>
            Press
          </li>
        </ul>
        <hr
          class="sc-f5137ace-1 ieaOSs"
        />
        <div
          class="sc-f5137ace-4 cVmtEU"
        >
          <div
            class="sc-f5137ace-5 yqDKo"
          >
            Join the mission to fund public goods.
          </div>
          <div
            class="sc-f5137ace-6 gSJNRe"
          >
            <button
              class="sc-21c50e09-2 bbXbsp undefined "
            >
              <span>
                Donate
              </span>
            </button>
            <button
              class="sc-21c50e09-2 bbXbsp undefined "
            >
              <span>
                Sign In
              </span>
            </button>
          </div>
        </div>
        <hr
          class="sc-f5137ace-1 bJA-dNh"
        />
        <div
          class="sc-697e6c26-0 kswkdX"
        >
          <a
            class="sc-697e6c26-1 ZHgeb"
            href="https://twitter.com/artizenfund"
            target="_blank"
          >
            twitter
          </a>
          <a
            class="sc-697e6c26-1 jyPDCi"
            href="https://artizen.link/discord"
            target="_blank"
          >
            discord
          </a>
          <a
            class="sc-697e6c26-1 eVSbPL"
            href="https://github.com/artizen-fund"
            target="_blank"
          >
            github
          </a>
          <a
            class="sc-697e6c26-1 ddRiBr"
            href="https://artizen.link/telegram"
            target="_blank"
          >
            telegram
          </a>
        </div>
      </footer>
    </div>
    <div
      class="sc-9199b4ac-0 brREMd sc-f5137ace-8 bjLigi"
    >
      <footer
        class="sc-f5137ace-7 gEbPfu"
      >
        <div>
          Open source platform made with ❤️ by a globally distributed team
        </div>
        <ul>
          <li>
            Privacy Policy
          </li>
          <li>
            Terms & Conditions
          </li>
        </ul>
      </footer>
    </div>
  </div>
</div>
`;
